### PR TITLE
updated docs for 0.8.0.preview.4

### PR DESCRIPTION
**Pull request by aryan-mann**
# Peer Review Information:
- Updated badge from 2020.2 to 2020.3
- Updated images & added a step for the new seed option in USim window.

Decided not to include the note about random seed and determinism in the tutorial. Perhaps we should plan to move the FAQ page to GitHub or have a wiki?

### CODE REVIEW

File: com.unity.perception/Documentation~/Tutorial/Phase3.md [hunk 1]
```diff
@@ -77,7 +77,8 @@ In order to make sure our builds are compatible with Unity Simulation, we need t
 
 Here, you can specify a name for the run, the number of Iterations the Scenario will execute for, and the number of Instances (number of nodes the work will be distributed across) for the run. This window automatically picks the currently active Scene and Scenario to run in Unity Simulation.
 
-* **:green_circle: Action**: Name your run `FirstRun`, set the number of Iterations to `1000`, and Instances to `20`. 
+* **:green_circle: Action**: Name your run `FirstRun`, set the number of Iterations to `1000`, and Instances to `20`.
+* **:green_circle: Action**: Click `Randomize` to generate a seed for your runs.
```
Comment (mkamalza) [inline com.unity.perception/Documentation~/Tutorial/Phase3.md:81]:
```suggestion
* **:green_circle: Action**: If you'd like to use a new random seed for this run of your Scenario, click `Randomize` to generate a new seed.
```

**Review by mkamalza — CHANGES_REQUESTED** [review]
So apart from the images, the run id and project id for this sample run is mentioned a couple of times in the instructions below. Can you please replace all of those with the new ids from the screenshots?

**Comment by mkamalza** [comment]
The thing with the seed explanation is OK, we will be adding the faq to github.

File: com.unity.perception/Documentation~/Tutorial/Phase3.md (hunk 1 above)
Comment (aryan-mann) [reply com.unity.perception/Documentation~/Tutorial/Phase3.md]: Nice catch, updating execution ID and project ID to match screenshots

**Review by mkamalza — APPROVED** [review]
LGTM!

**Pull request merged**